alpha-blog/app/views/layouts/_navigation.html.erb

```diff
@@ -20,6 +20,19 @@
             <%= link_to "View articles", articles_path, class: "dropdown-item" %>
           </div>
         </li>
+
+        <li class="nav-item dropdown">
+          <div class="nav-link dropdown-toggle" id="navbarDropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
```

Comment:
The best practice for CSS classes & IDs is kebab-case. Please change

Reply:
but then this is bootstrap classes and ID tho. Then I'll have to copy the styling over?

Reply:
Then nvm